plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -16,7 +16,22 @@
 #include "synapticsmst-device.h"
 #include "synapticsmst-common.h"
 
-#define BLOCK_UNIT		64
+#define MSTDBG 	0 
```

Comment:
I don't really like this; what's the purpose? Debug messages are turned off by default when running the daemon normally, so you can just use g_debug without worrying.

Reply:
Yeah I agree.  And if the debugging is too verbose for regular daemon debug (such as raw comms), it can be hidden in plugin specific debug.